Drop-In/src/main/java/com/braintreepayments/api/SelectPaymentMethodParentFragment.java

```diff
@@ -0,0 +1,124 @@
+package com.braintreepayments.api;
+
+import android.os.Bundle;
+import android.util.TypedValue;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.VisibleForTesting;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentResultListener;
+import androidx.viewpager2.widget.ViewPager2;
```

Comment:
:interesting:

Reply:
Yeah they already had a `ViewPager` class in the support lib I think. The naming is odd but it's a way for the googs to prevent breaking changes at least.